server/db/db.js

```diff
@@ -84,10 +97,53 @@ async function execute(path, params = {}) {
   sql = sql.replace(/\$\{[^{}]+\}/g, queryParam);
   const values = queryVariables ? queryVariables.map(p => params[p]) : [];
   return query(sql, values);
-}
+};
+
+const migrate = async () => {
+  let existingMigrations = [];
+  try {
+    const result = await pool.query('SELECT * FROM migrations');
```

Comment:
despite it being small, we can utilize our new `execute` function here:

```suggestion
    const result = await db.execute('server/sql/migration_queries/get_all.sql');
```

and you'll have to make a `migration_queries` (or whatever you want to name it) directory with that file in it